provisioning/local/aws.tf

```diff
@@ -0,0 +1,88 @@
+provider "aws" {
+  region  = local.region
+  profile = var.aws_profile
+}
+
+locals {
+  region = "eu-central-1"
+}
+
+variable "aws_profile" {
+  type = string
+  validation {
+    condition     = length(var.aws_profile) > 0
+    error_message = "The \"aws_profile\" must be non-empty string."
+  }
+}
+
+resource "aws_iam_user" "api_error_control_iam_user" {
+  name          = "${var.name_prefix}-api-error-control-User"
+  force_destroy = true
+}
+
+resource "aws_iam_access_key" "api_error_control_credentials" {
+  user = aws_iam_user.api_error_control_iam_user.id
+}
+
+resource "aws_s3_bucket" "api_error_control_s3_bucket" {
```

Comment:
Tady bych navrhoval spis treba nazev `logs_bucket` misto `api_error_control_s3_bucket`. Kdyz to pak nekde budes referencovat, budes hledat bucket na logy, ne S3 bucket pro Api Error Control.

Obecne, pokud ma ten resource enjaky specificky vyznam, je lepsi ho pojmenovavat podle toho. Viz. napr. https://github.com/keboola/sandboxes-service/blob/4f51346d62ab0121f092a618e0ddeab1ec2707b4/provisioning/local/aws_logs.tf#L1